src/main/java/com/cqse/teamscaleupload/resolve/ReportPatternUtils.java

```diff
@@ -21,7 +23,7 @@
 public class ReportPatternUtils {
 
     /** Expected pattern for formats in the input file, matches e.g. "[vs_coverage]" */
-    private static final Pattern FORMAT_PATTERN = Pattern.compile("\\[(\\w+)\\]");
+    private static final Pattern FORMAT_PATTERN = Pattern.compile("\\[(\\w+)]");
```

Comment:
Interesting, I was not aware that the closing bracket does not need escaping in a regex: https://stackoverflow.com/questions/43044918/why-closing-square-bracket-doesnt-require-escaping-in-java-regex

Thanks for noticing!